net/shorewall/Makefile

```diff
@@ -0,0 +1,118 @@
+#
+# Copyright (C) 2008-2012 OpenWrt.org
+# Copyright (C) 2017 Willem van den Akker <wvdakker@wilsoft.nl>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=shorewall
+PKG_VERSION:=5.1.4.1
+PKG_DIRECTORY:=5.1
+PKG_MAINVERSION:=5.1.4
+PKG_RELEASE:=2
```

Comment:
no need to bump the release as is not in tree at the moment

Reply:
It depends who is reviewing ;)

Reply:
New package -> PKG_RELEASE:=1